components/CreateNewPost/index.jsx

```diff
@@ -30,7 +30,7 @@ const CreateNewPost = () => {
   };
 
   return (
-    <StyledSection rows="auto 1fr" colums="1fr" padding="25px">
+    <StyledSection jsc="center" rows="auto 1fr" colums="1000px" padding="25px">
```

Comment:
- The absolut value `colums 1000px ` will break your layout on the `create-post page` **on mobile view.**
Try to use relative units to avoid the problem. 
- There is a typo in columns. If you correct this, don't forget to adjust the corresponding code in the `StyledSection `component. ✌️
- try to change the prop name `jsc` in something more descriptive  

```suggestion
    <StyledSection jsc="center" rows="auto 1fr" columns="100%" padding="25px">
```